app/forms/ctc/base_primary_filer_form.rb

```diff
@@ -9,6 +9,10 @@ class BasePrimaryFilerForm < QuestionsForm
     validates :primary_ssn, social_security_number: true, if: -> { ["ssn", "ssn_no_employment"].include? primary_tin_type }
     validates :primary_ssn, individual_taxpayer_identification_number: true, if: -> { primary_tin_type == "itin" }
 
+    before_validation do
```

Comment:
I feel like we're going to want to make sure this is consistent across all implementations of SSN persistance. Maybe we can have the SSN/ITIN validator handle scrubbing when validating, and then add the removal to the before_save on intake when these fields are present. 

(I also thought that when we discussed length of 11 SSN fields, Travis said that this scrubbing was already being done somewhere consistently to ensure only numbers got into the DB?)